src/components/VoteRow.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+const Container = styled.div`
+  display: flex;
+  justify-content: center;
+`;
+const Wrapper = styled.div`
+  width: 150px;
+  margin: 10px;
+`;
+const Button = styled.button`
+  width: 100px;
+`;
+
+export default function VoteRow({ candidate, order, setCandidates }) {
+  const { id } = candidate;
+  const vote = async () => {
+    try {
+      const result = await axios.get(
+        `http://ec2-3-34-5-220.ap-northeast-2.compute.amazonaws.com:2020/vote?id=${id}`
+      );
+      alert('vote success');
+    } catch (e) {
+      console.log(e);
+      alert('vote fail');
+    }
+  };
```

Comment:
1. `result`변수가 할당되었지만 사용되지 않고 있어요! 굳이 변수 선언할 필요는 없을 것 같아요~
2. `candidate`에서 뽑아온 `name`으로 누구에게 투표했는지 알려주세용

```suggestion
    try {
      await axios.get(
        `http://ec2-3-34-5-220.ap-northeast-2.compute.amazonaws.com:2020/vote?id=${id}`
      );
      alert( `${name}님에게 투표 완료!`);
    } catch (e) {
      console.log(e);
      alert('vote fail');
    }
  };
```